src/spectrecoff/LiveDisplay.fs

```diff
@@ -0,0 +1,36 @@
+﻿[<AutoOpen>]
+module SpectreCoff.LiveDisplay
+
+open System.Threading.Tasks
+open Spectre.Console
+
+type Configuration =
+    { AutoClear: bool Option
```

Comment:
Not a big fan of bool Options in general. I mean, it's either true or false, right?

Reply:
That is a very good point actually, makes little sense :) I will keep it in mind